ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/block/BlockImporterTest.java

```diff
@@ -313,9 +313,19 @@ public void importBlock_parentBlockFromSameSlot() throws Exception {
   }
 
   private void assertSuccessfulResult(final BlockImportResult result) {
+    assertSuccessfulResult(result, true);
```

Comment:
if the import result is false, wouldn't that imply notifications aren't sent?
I was thinking this would be something like `assertSuccessfulResult(result, result.isSuccessful())`

Reply:
Known blocks are still deemed successful, but they not necessarily need to be sent since they have already been imported and notification has been sent for them.